content/en/docs/setup/additional-setup/pod-security-admission/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Install Istio with Pod Security Admission
+description: Install and use Istio with the Pod Security admission controller..
+weight: 70
+aliases:
+    - /docs/setup/kubernetes/install/pod-security-admission
+    - /docs/setup/kubernetes/additional-setup/pod-security-admission
+keywords: [psa]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+Follow this guide to install, configure, and use an Istio mesh with the Pod Security admission controller
+([PSA](https://kubernetes.io/docs/concepts/security/pod-security-admission/)) enforcing the `baseline` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/) on namespaces in the mesh.
+
+By default Istio injects an init container, `istio-init`, in pods deployed in
+the mesh. The `istio-init` requires the user or
+service-account deploying pods to the mesh to have sufficient Kubernetes RBAC
+permissions to deploy [containers with the `NET_ADMIN` and `NET_RAW` capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container).
+
+However, the `baseline` policy does not include `NET_ADMIN` or `NET_RAW` in its [allowed capabilities](https://kubernetes.io/docs/concepts/security/pod-security-standards/#baseline). Therefore, it is necessary to use Istio mesh with the [Istio Container Network Interface plugin](/docs/setup/additional-setup/cni/). The `istio-cni-node` DaemonSet in the `istio-system` namespace requires `hostPath` volumes to access local CNI directories. Since this is not allowed in the `baseline` policy, the `istio-system` namespace needs to enforce the `privileged` [policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/#privileged).
+
+## Install Istio with PSA
+
+1. Create the `istio-system` namespace and label it to enforce the `privileged` policy.
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    $ kubectl label --overwrite ns istio-system \
```

Comment:
some profiles deploy CNI in other namespaces. I think openshift is in openshift-operators, and on GKE in kube-system.

Do you know if kube-system is implicitly `pod-security.kubernetes.io/enforce=privileged`?

Reply:
So Kubernetes recommends exempting the kube-system namespace from the PSA controller entirely to avoid giving it privilege since that would include such a large scope:

See [step 2](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/#set-modes-versions-and-standards)

Reply:
Cool, it may be noted that you only need to make isti-system privileged if CNI is deployed there then? https://github.com/istio/istio.io/pull/12975#pullrequestreview-1369736339 also would implicitly do that I think